### PR TITLE
Fix image build to fail when runtime deps fail to download

### PR DESCRIPTION
**Pull request by rjew**
* When runtime deps fail to download, the image build will now report as `ERROR`
* Fix issues in function build phase
  - Comparisons now work as intended
  - If there is a `HANDLER` arg, the rest of the `RUN` step is skipped

### CODE REVIEW

File: function-template/Dockerfile [hunk 1]
```diff
@@ -8,10 +8,10 @@ COPY . .
 
 RUN echo -n ${HANDLER} > /tmp/handler && \
     [ -n "${HANDLER}" ] || \
-    ([ "$(ls *.clj | wc -l)" == "1" ] || (echo "Exactly 1 source file needed when HANDLER is empty" >&2 && exit 1)) && \
+    (([ "$(ls *.clj | wc -l)" -eq "1" ] || (echo "Exactly 1 source file needed when HANDLER is empty" >&2 && exit 1)) && \
```
Comment (imikushin) [inline function-template/Dockerfile:11]: What's wrong with `==`?

**Review by imikushin — APPROVED** [review]
Looks good!

File: function-template/Dockerfile (hunk 1 above)
Comment (rjew) [reply function-template/Dockerfile:11]: In this specific case either `==` or `-eq` works but `-eq` is better for numeric comparisons. This also makes it more consistent with our other base images.

Comment (imikushin) [reply function-template/Dockerfile:11]: Thanks! Merge away!

**Pull request merged**